### ISSUE TITLE
Error Running Quasar Vite plugin with Vitest: Unexpected token

### ISSUE THREAD

**Issue by ChronosMasterOfAllTime** [quasarframework/quasar#15997]
### What happened?

Getting unexpected token when extending a Quasar component during tests.

```
FAIL  src/App.spec.ts [ src/App.spec.ts ]
Error: Parse failure: Unexpected token (16:29)
At file: /src/components/global/extensions/ExtendedBadge.vue
Contents of line 16:   const _component_q_badge = ;
```

### What did you expect to happen?

Tests should run successfully.

### Reproduction URL

https://stackblitz.com/edit/quasarframework-s7m4nh

### How to reproduce?

1. Go to repro link
2. Look at console after tests complete running

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar), Plugins (quasar)

### Platforms/Browsers

Other

### Quasar info output

```Shell
N/A
```

### Relevant log output

```Shell
FAIL  src/App.spec.ts [ src/App.spec.ts ]
Error: Parse failure: Unexpected token (16:29)
At file: /src/components/global/extensions/ExtendedBadge.vue
Contents of line 16:   const _component_q_badge = ;
```

### Additional context

This is happening when a component uses the `extends` clause within the OPTIONS API of Vue.

**Comment by freddy38510** [comment]
I've just made a few changes that seem to work https://stackblitz.com/edit/quasarframework-ks2yeg?file=vite.config.ts

`@quasar/vite-plugin` has some missing checks for test mode. I'll create a PR tomorrow.

**Comment by ChronosMasterOfAllTime** [comment]
> I've just made a few changes that seem to work https://stackblitz.com/edit/quasarframework-ks2yeg?file=vite.config.ts
> 
> `@quasar/vite-plugin` has some missing checks for test mode. I'll create a PR tomorrow.
> 

Awesome; thanks for getting to this so quickly!

**Comment by rstoenescu** [comment]
Fix will be available in `@quasar/vite-plugin` v1.4.0
